fix/mpassit/diaglist

```diff
@@ -1,4 +1,5 @@
 rainnc                   RAINNC
+frainnc                  ACFRAIN
```

Comment:
I think this needs the correct UPP version. Can we do it now? @EricJames-NOAA

Reply:
@guoqing-noaa if the var 'frainnc' is in the MPAS diag files, it should be fine to add this line here.

Reply:
> Thanks @EricJames-NOAA.
> 
> @chunhuazhou If dev1 has this line and you have updated the MPAS-Model to the same as dev1, we should be able to see `frainnc` in the MPAS diag files, right?

Actually mpassit in dev3 couldn't handle frainnc and failed, probably due to different mpassit versions in dev1 and dev3. I reverted the change to mpassit/diaglist so we have no changes to diaglist for now.